templates/CRM/Profile/Form/Dynamic.tpl

```diff
@@ -148,7 +148,7 @@
               {elseif $n|substr:0:5 eq 'phone'}
                 {assign var="phone_ext_field" value=$n|replace:'phone':'phone_ext'}
                 {$form.$n.html}
-                {if $form.$phone_ext_field.html}
+                {if array_key_exists($phone_ext_field, $form)}
```

Comment:
I guess I didn't try saving before. It seems something happens where $form doesn't exist on save. It saves, but then the screen fills with warnings (which happened before), and now with this addition it also crashes.

I suppose those are two different issues. We could punt the second one by doing `{if $form && array_key_exists($phone_ext_field, $form)}`, which will generate yet another warning, but shouldn't crash.

Reply:
Actually that doesn't make sense to do - we're just swapping one warning for another. Hmm.

Reply:
hmm. let's park it and I'll have another look later. Thanks heaps.